core/src/epicli/data/common/ansible/playbooks/roles/sshtunnel/files/tunnel.sh

```diff
@@ -1,12 +1,48 @@
 #!/bin/bash
```

Comment:
This will work across bigger number of operating systems! (for example in NixOS ;P)
```suggestion
#!/usr/bin/env bash
```